crowdsourcer/static/js/volunteer.esm.js

```diff
@@ -4,12 +4,17 @@ async function getAvailableAuthorities($fs) {
     var $session = $fs.find('.field_session').eq(0);
     var $section = $fs.find('.field_section').eq(0);
     var $rt = $fs.find('.field_rt').eq(0);
+    var $id = $fs.find('[name$="-id"]').eq(0);
 
     if ($rt.val() == "" || $section.val() == "") {
       return { results: [] };
     }
 
-    url = url + "?rt=" + $rt.val() + "&s=" + $section.val() + "&ms=" + $session.val()
+    url = url + "?rt=" + $rt.val() + "&s=" + $section.val() + "&ms=" + $session.val();
```

Comment:
FWIW jQuery has a built-in function to construct query strings like this – `$.param()`, which has the advantage of _also_ URL-encoding the keys and values, so they’re safe to append to a URL. eg:

```js
url = url + '?' + $.param({
  rt: $rt.val(),
  s: $section.val(),
  ms: $session.val()
});
```

You could _even_ store everything in a variable at first, then conditionally add your `id` item (below), and _then_ parametrize them all in one go at the end, eg:

```js
var params = {
  rt: $rt.val(),
  s: $section.val(),
  ms: $session.val()
};

if (…) {
  params["id"] = $id.val();
}

url = url + '?' + $.param(params);
```

Or _even_ (replacing the need to declare `$session`, `$section`, and `$rt` up front, since you never use them again anyway):

```js

var params = {
  rt: $fs.find('.field_rt').eq(0).val(),
  s: $fs.find('.field_section').eq(0).val(),
  ms: $fs.find('.field_session').eq(0).val()
};

if (…) {
  params["id"] = $id.val();
}

url = url + '?' + $.param(params);
```

Just a thought!